spec/classes/plugin_container_spec.rb

```diff
@@ -13,7 +13,9 @@
             .that_subscribes_to('Class[Pulpcore::Install]')
             .that_notifies(['Class[Pulpcore::Database]', 'Class[Pulpcore::Service]'])
           is_expected.to contain_package('pulpcore-plugin(container)')
-          is_expected.to contain_concat__fragment('plugin-container').with_content("\n# container plugin settings\nTOKEN_AUTH_DISABLED=True")
+          is_expected.to contain_concat__fragment('plugin-container')
+            .with_content("\n# container plugin settings\nTOKEN_AUTH_DISABLED=True")
+            .with_content(/^FLATPAK_INDEX=True/)
```

Comment:
The first expectation is a complete string and that no longer matches. It probably needs to be
```suggestion
            .with_content("\n# container plugin settings\nTOKEN_AUTH_DISABLED=True\nFLATPAK_INDEX=True")
```